index.html

```diff
@@ -1,13 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <title>Incremental and Decremental Support Vector Machine Learning</title>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    
+      <h1>Incremental and Decremental Support Vector Machine Learning</h1>
+    <hr/>
```

Comment:
Careful. Anything you put into the `<head></head>` section won't be displayed on the page. When the requirements asked you to add a title, it was hinting at using the `<title></title>` element